server/chat-plugins/daily-spotlight.ts

```diff
@@ -256,7 +256,7 @@ export const commands: Chat.ChatCommands = {
 	},
 };
 
-export const onRenameRoom: Rooms.RenameHandler = (oldID, newID) => {
+export const onRenameRoom: Chat.Handlers.RenameRoom = (oldID, newID) => {
```

Comment:
What if:

```ts
export const handlers: Chat.Handlers = {
  onRenameRoom(oldID, newID) {
  }
};
```

Would provide some more safety, too!

Reply:
Although also consider "listeners" or "hooks"

Reply:
Done. Renamed to `hooks` as well.

Reply:
I should probably not have suggested "hooks" since it's my least favorite of the three. I like "listeners" or "handlers" better.

Reply:
Or even "events".

Reply:
Alright. Personally, I prefer hooks since it's shortest, but handlers will do?

Reply:
Done.

Reply:
Yeah, handlers is probably best.